backend/api/image_generation.py

```
@@ -0,0 +1,300 @@
+            
+            # Return temporary URL directly (no local storage)
+            return temp_image_url
+            
+        except Exception as e:
+            debug_log(f"[ERROR] Image generation failed for scene '{scene_title}': {str(e)}")
+            return ""  # Return empty string on failure
+
+
+async def generate_scenes_with_images(
+    scenes: List[Dict[str, Any]], 
+    session_id: str = None
```

Ruff (0.14.2)

Unused function argument: `session_id`

(ARG001)